open_instruct/ground_truth_utils.py

```diff
@@ -663,8 +672,8 @@ async def async_call(
         for attempt in range(max_retries):
             # judges the quality of a response
             try:
-                system_prompt = "Do not generate text between the <think> and </think> tags."  # "You are a concise assistant who gives very short explanations before giving a quality score."
-                messages = build_messages(prompt, system_prompt)
+                # system_prompt = "Do not generate text between the <think> and </think> tags."  # "You are a concise assistant who gives very short explanations before giving a quality score."
+                messages = build_messages(prompt, system_prompt=None)
```

Comment:
```suggestion
                messages = build_messages(prompt)
```
Since in `build_messages` the system_prompt arg is optional anyway.